ndbench-dynamodb-plugins/build.gradle

```diff
@@ -8,6 +8,8 @@ repositories {
 dependencies {
     compile project(':ndbench-api')
     compile "com.amazonaws:aws-java-sdk-dynamodb:latest.release"
+    compile "com.amazonaws:aws-java-sdk-cloudwatch:latest.release"
```

Comment:
are both necessary? or only one?

Reply:
I do not believe we need the cloudwatch. I do not recall adding it.